### PR TITLE
Apply job rate limit only to mqtt based jobs

### PR DESCRIPTION
**Pull request by adamlazik1**
CC @adamruzicka 

Note: last line is displaying change because of the added newline.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

**Comment by adamlazik1** [comment]
SME approved.

**Comment by adamlazik1** [comment]
Could someone review for wording?

File: guides/common/modules/proc_setting-the-job-rate-limit-on-smartproxy.adoc [hunk 1]
```diff
@@ -4,6 +4,8 @@
 You can limit the maximum number of active jobs on a {SmartProxy} at a time to prevent performance spikes.
 The job is active from the time {SmartProxy} first tries to notify the host about the job until the job is finished on the host.
 
+Job rate limit only applies to mqtt based jobs.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_setting-the-job-rate-limit-on-smartproxy.adoc:7]:
```suggestion
The job rate limit only applies to mqtt based jobs.
```

**Review by maximiliankolb — APPROVED** [review]

**Comment by adamlazik1** [comment]
Thank you @maximiliankolb! Will merge once the check completes.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   30ca6a3eb..b3013c474  3.5 -> 3.5
   5557b197f..056ab5fdb  3.6 -> 3.6

**Pull request merged**